dist/macos/bundle/build_dist.sh.in

```diff
@@ -34,24 +34,29 @@ fi
 info "Copying binaries into bundle"
 # Copy the folder instead of globbing unquoted path
 cp -r "$B_BINDIR" "$B_MACOS" || exit 1
-
-# Check for macdeployqt on MacPorts
-if which -s port ; then
-    info "MacPorts found, searching for macdeployqt"
-    DEPLOYQT="$(port contents qt5-qttools | grep --only --max-count 1 '/.*macdeployqt')"
-    if [ ! -x "$DEPLOYQT" ]; then
-        error "Please install package qt5-qttools"
-        exit 1
+if [ -f "/Users/runner/Qt/5.15.2/clang_64/bin/macdeployqt" ]; then
+    DEPLOYQT="/Users/runner/Qt/5.15.2/clang_64/bin/macdeployqt"
+elif [ -f "/Users/runner/Qt/6.6.0/clang_64/bin/macdeployqt" ]; then
+    DEPLOYQT="~/Qt/6.6.0/clang_64/bin/macdeployqt"
+else
+    # Check for macdeployqt on MacPorts
+    if which -s port ; then
+        info "MacPorts found, searching for macdeployqt"
+        DEPLOYQT="$(port contents qt5-qttools | grep --only --max-count 1 '/.*macdeployqt')"
+        if [ ! -x "$DEPLOYQT" ]; then
+            error "Please install package qt5-qttools"
+            exit 1
+        fi
     fi
-fi
 
-# Check for macdeployqt on Homebrew
-if which -s brew ; then
-    info "Homebrew found, searching for macdeployqt"
-    DEPLOYQT="$(brew list qt@5 | grep --only '/.*macdeployqt' | head -1)"
-    if [ ! -x "$DEPLOYQT" ]; then
-        error "Please install package qt"
-        exit 1
+    # Check for macdeployqt on Homebrew
+    if which -s brew ; then
+        info "Homebrew found, searching for macdeployqt"
+        DEPLOYQT="$(brew list qt@5 | grep --only '/.*macdeployqt' | head -1)"
+        if [ ! -x "$DEPLOYQT" ]; then
+            error "Please install package qt"
+            exit 1
+        fi
     fi
 fi
 
```

Comment:
We can't hardcode paths here. This breaks things for developers. It only benefits the CI.

Can we auto-discover these paths? I don't have a Mac, so I'm fairly unsure here.